### PR TITLE
Change plat to platform for disable feature

### PR DESCRIPTION
**Pull request by renfeiw**
Signed-off-by: renfeiw <renfeiw@ca.ibm.com>

### CODE REVIEW

**Comment by renfeiw** [comment]
internal build: Grinder_TKG/1220

**Comment by renfeiw** [comment]
run aqa --sdk_resource nightly --build_list openjdk --target sanity.openjdk --jdk_version 11 --jdk_impl hotspot --platform x86-64_linux

File: src/org/testKitGen/TestInfoParser.java [hunk 1]
```diff
@@ -190,14 +190,17 @@ private void parseDisableInfo(TestInfo ti) {
 			String impl = getDisabledEle(disabled, "impl", ti.getTestCaseName());
 			String vendor = getDisabledEle(disabled, "vendor", ti.getTestCaseName());
 			String version = getDisabledEle(disabled, "version", ti.getTestCaseName());
-			String plat = getDisabledEle(disabled, "plat", ti.getTestCaseName());
+			String platform = getDisabledEle(disabled, "platform", ti.getTestCaseName());
+			if (platform != null) {
+				platform = getDisabledEle(disabled, "plat", ti.getTestCaseName());
```
Comment (karianna) [inline src/org/testKitGen/TestInfoParser.java:195]: Is that to support backwards compatibility?

**Review by karianna — CHANGES_REQUESTED** [review]

File: src/org/testKitGen/TestInfoParser.java (hunk 1 above)
Comment (renfeiw) [reply src/org/testKitGen/TestInfoParser.java:195]: Thanks for catching it. I mistakenly had `!=` instead of `==`, updated the change.

**Review by karianna — APPROVED** [review]
LGTM

**Comment by renfeiw** [comment]
internal grinder: Grinder_TKG/1228

**Review by smlambert — APPROVED** [review]

**Pull request merged**